pages.de/common/git-config.md

```diff
@@ -0,0 +1,37 @@
+# git config
+
+> Verwalten von benutzerdefinierten Optionen für Git Repositories.
+> Diese Optionen können lokal (für das aktiven Repository) or global (für den aktiven Benutzer) sein.
+> Weitere Informationen: <https://git-scm.com/docs/git-config/de>.
```

Comment:
Thanks for your contribution and welcome to tldr. Is the `de` specifier at last necessary, I think the site would redirect to the correct locale based on your system/browser's language preference.

```suggestion
> Weitere Informationen: <https://git-scm.com/docs/git-config>.
```

Reply:
I think it is. For me (german language preference set), it does not redirect to the german site.

Reply:
> I think it is. For me (german language preference set), it does not redirect to the german site.

Thanks for verifying it, I tried now. I can confirm the same.